### DISCUSSION TITLE
Relation to https://github.com/ElinamLLC/SharpVectors

### DISCUSSION THREAD

**Discussion by ReIaxo** [dotnetprojects/SVGImage#93] [General]
Hi, I see that the two projects mention each other sometimes. What are the differences and relations and which one should I use when I want to display svg in WPF?

**Comment by jogibear9988** [comment]
At first they had no relation. I used a old codeproject project to generate this lib. 
But as I don't use this any more (we switched to a html ui) the maintainer of https://github.com/ElinamLLC/SharpVectors @paulushub also started to fix some things in this lib.

Wich you use, it depends. I think if you only need a specialized image control to display a SVG, use this. If you need to interact with the SVG, access the DOM, ... use the other one.

Also, not every SVGs will work, there is only very very limited animation support.
